### PR TITLE
CDDA Emphasis

### PR DESCRIPTION
**Pull request by privatezero**
Adds some commands for querying and compensating for CD Emphasis

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2479,6 +2496,21 @@ <h3>Rip a CD with Cdda2wav</h3>
...
       </div>
       <!-- ends Rip with CDDA2WAV -->
+
+      <!-- Check for CD Emphasis -->
+        <label class="recipe" for="cd-emph-check">Check/Compensate for CD Emphasis</label>
+        <input type="checkbox" id="cd-emph-check">
+        <div class="hiding">
+          <h3>Check/Compensate for CD Emphasis</h3>
+          <p>While somewhat rare, certain CDs had 'emphasis' applied as a form of noise reduction. This seems to mostly affect early (1980s) era CDs and some CDs pressed in Japan. Emphasis is part of the <a href="https://en.wikipedia.org/wiki/Compact_Disc_Digital_Audio#Standard">Red Book standard</a> and, if present, must be compensated for to ensure accurate playback. CDs that use emphasis contain flags on tracks that tell the CD player to de-emphasize the audio on playback. When ripping a CD with emphasis, it is important to take this into account and either apply de-emphasis while ripping, or if storing a 'flat' copy, create another de-emphasized listening copy.</p>
+
```
Comment (retokromer) [inline index.html]: delete useless empty line

**Review by retokromer — APPROVED** [review]
one nit, otherwise LGTM

**Review by kieranjol — APPROVED** [review]
This is great, thank you!

**Review by kfrn — APPROVED** [review]
Cheers! I don't have the chance to actually test the commands, but the content looks great!

**Review by retokromer — APPROVED** [review]

**Comment by privatezero** [comment]
Thanks guys! This seemed obscure, but important for preservation/mass reformatting projects! (For example - sounds to my ears like Spotify did not de-emphasize their copy of my test CD and the high hats are obnoxious! - maybe I will ping AVAA about adding a section about EQ similar to the noise reduction section)

**Comment by kieranjol** [comment]
Obnoxious hihats!

**Pull request merged**